packages/site/cms-site/src/blocks/VimeoVideoBlock.module.scss

```diff
@@ -0,0 +1,19 @@
+.videoContainer {
+    overflow: hidden;
+    position: relative;
+    aspect-ratio: var(--aspect-ratio);
+}
+
+.videoContainer.fill {
```

Comment:
Why is this different to DAM video block, where fill is standalone?

Reply:
I changed this later on because the first class is unnecessary. I must have missed Vimeo 81d22cf5b37429b8e6d6a6e51b7cc283a55040a4